src/screens/OrgAdminMemberPage/OrgAdminMemberPage.tsx

```diff
@@ -1,15 +1,35 @@
 import React from 'react';
 import AdminNavbar from 'components/AdminNavbar/AdminNavbar';
 import styles from './OrgAdminMemberPage.module.css';
+import UserCard from 'components/UserCard/UserCard';
+import { useQuery } from '@apollo/client';
+import { MEMBERS_LIST } from 'GraphQl/Queries/Queries';
 
 function OrgAdminMemberPage(): JSX.Element {
+  const a = window.location.href.split('=')[1];
+
+  const b = '/orghome/id=' + a;
```

Comment:
"a" and "b" are **not** descriptive variable names, please make use of descriptive names when declaring your variables